reactive-client/build.gradle

```diff
@@ -0,0 +1,57 @@
+buildscript {
+    repositories {
+        jcenter()
+        google()
+        maven {
+            url 'http://nexus.gluonhq.com/nexus/content/repositories/releases'
+        }
+    }
+    dependencies {
+        classpath 'org.javafxports:jfxmobile-plugin:2.0.28'
```

Comment:
Are we using Gluon VM for this sample? All the rest of the samples (but `singleview-gluonvm`) use 1.3.+.

Reply:
Not sure... is using Gluon VM a bad thing? Can switch if needed

Reply:
I'm not saying it is a bad thing of course. We'll have to migrate all the samples when GVM it is released. But so far we have them using the former plugin, so that's why I asked.

Anyway, I can't deploy the sample to iOS with GVM, while it works fine with 1.3.12:
```[14:24:01:596] : Progress: VerifyingApplication [40%] 
[14:24:01:857] : Error: APIInternalError, Description: Failed to unhide archs in executable file:///private/var/installd/Library/Caches/com.apple.mobile.installd.staging/temp.eE0ADH/extracted/reactive-client.app/RxGluonApp                                    
```